### PR TITLE
refactor OS-specific menu-items

### PR DESCRIPTION
**Pull request by luclu**
closes https://github.com/ethereum/mist/issues/1168

Tested on win, linux, mac

### CODE REVIEW

File: interface/i18n/mist.en.i18n.json [hunk 1]
```diff
@@ -71,7 +71,8 @@
                 "toFront": "Bring All to Front"
             },
             "help": {
-                "label": "Help"
+                "label": "Help",
+                "reportBug": "Report a bug on Github"
```
Comment (evertonfraga) [inline interface/i18n/mist.en.i18n.json]: i created that reportbug thing. Maybe we should rephrase to "Report an issue on Github"?

Comment (luclu) [inline interface/i18n/mist.en.i18n.json]: done! 👍

File: modules/menuItems.js [hunk 2]
```diff
@@ -482,19 +487,45 @@ var menuTempl = function(webviews) {
...
+            {
+                label: i18n.t('mist.applicationMenu.app.checkForUpdates'),
+                click: function() {
+                    updateChecker.runVisibly();
                 }
-            }]
+            }
+        );
+    }
+    if(process.platform === 'darwin') {
```
Comment (evertonfraga) [inline modules/menuItems.js]: @luclu is there a reason why is this enabled only on Mac?

Comment (evertonfraga) [inline modules/menuItems.js]: anyways, I removed this restriction and tested on Linux+Mac. all worked

**Comment by evertonfraga** [comment]
merged on #1269.

**Pull request closed without merging**